diff -urN lstat-2.2.orig/Makefile.in lstat-2.2/Makefile.in
--- lstat-2.2.orig/Makefile.in	Tue May 21 12:17:34 2002
+++ lstat-2.2/Makefile.in	Tue May 21 12:59:12 2002
@@ -44,7 +44,7 @@
 
 all: clean_CVS $(PROGS) $(LIBS) $(MISC) templates/Makefile
 
-install: install_files initrc update_apache_conf mkgraphs
+install: install_files initrc update_apache_conf
 
 install_files:
 	@echo "Instaling  library files..."
@@ -65,9 +65,9 @@
 	echo ;\
 	fi;
 
-	@${INSTALL} -p -m 700 -g $(WWWGROUP) -o $(WWWUSER) -d $(DESTDIR)${OBJECTSDIR}
+	@${INSTALL} -p -m 700 -d $(DESTDIR)${OBJECTSDIR}
 	@echo "Instaling  pages and user config files..."
-	@${INSTALL} -p -m 700 -g $(WWWGROUP) -o $(WWWUSER) -d $(DESTDIR)${PAGESDIR}
+	@${INSTALL} -p -m 700 -d $(DESTDIR)${PAGESDIR}
 
 	@if [ ! -f $(DESTDIR)${PAGESDIR}index.pg ]; then \
 	${INSTALL} -m 644 ${SRCDIR}/pages/index.pg $(DESTDIR)${PAGESDIR} ;\
@@ -97,7 +97,7 @@
 	@${INSTALL} -p -m 644 ${SRCDIR}/images/skins/sky/uni/* $(DESTDIR)${WWWDIR}/skins/sky/uni
 
 	@echo "Instaling CGI scripts"
-	@${INSTALL} -p -m 700 -g $(WWWGROUP) -o $(WWWUSER) -d $(DESTDIR)${WWWDIR}statimg
+	@${INSTALL} -p -m 700 -d $(DESTDIR)${WWWDIR}statimg
 	@${INSTALL} -d $(DESTDIR)${WWWDIR}edit
 	@${INSTALL} -p -m 755 ${SRCDIR}/lstat.cgi $(DESTDIR)${WWWDIR}
 	@${INSTALL} -p -m 755 ${SRCDIR}/edit.cgi $(DESTDIR)${WWWDIR}edit
@@ -144,11 +144,6 @@
 	echo "ok";\
 	fi;
 
-mkgraphs:
-	@echo "Creating some stats"
-	@$(BINDIR)/Mkgraph.pl
-	@echo
-
 clean:
 	cd $(SRCDIR) && rm -f $(PROGS)
 	cd $(SRCDIR)/lib && rm -f $(LIBS)
