diff -Nur lstat-2.2.orig/src/lstatd.in lstat-2.2/src/lstatd.in
--- lstat-2.2.orig/src/lstatd.in	Wed Apr 10 13:56:59 2002
+++ lstat-2.2/src/lstatd.in	Tue Feb 11 13:49:14 2003
@@ -22,8 +22,8 @@
 if ($Debugging < 3) {
 #start daemon
 chdir '/' or die "Can't chdir to /: $!";
-#umask 0077;
-umask 0;
+umask 0077;
+#umask 0;
 open STDIN, '/dev/null'   or die "Can't read /dev/null: $!";
 open (STDOUT, ">$logfile") or die "Can't write to $logfile: $!";
 open (STDERR, '>&STDOUT') or die "Can't redirect STDERR to STDOUT: $!";
diff -Nur lstat-2.2.orig/src/misc/lstat.conf.in lstat-2.2/src/misc/lstat.conf.in
--- lstat-2.2.orig/src/misc/lstat.conf.in	Sat Jan 26 16:54:59 2002
+++ lstat-2.2/src/misc/lstat.conf.in	Tue Feb 11 13:50:05 2003
@@ -6,6 +6,7 @@
 AddHandler cgi-script .cgi
 
 AllowOverride AuthConfig Limit
+DirectoryIndex lstat.cgi
 
 #setting expiration pngfiles
 #make sure if Expires mod is enabled

--- lstat-2.2/src/lib/config.in~	Fri Jan 25 19:06:29 2002
+++ lstat-2.2/src/lib/config.in	Mon May 26 02:29:35 2003
@@ -22,12 +22,12 @@
 PROC_INT_FILE=/proc/net/dev
 PROC_PART_FILE=/proc/partitions
 DF=@@df@@
-IPCHAINS=@@ipchains@@
-IPTABLES=@@iptables@@
+#IPCHAINS=/sbin/ipchains
+IPTABLES=/usr/sbin/iptables
 PROC_CPUINFO=/proc/cpuinfo
 UPTIME=@@uptime@@
 IFCONFIG=@@ifconfig@@
-REDHAT=/etc/redhat-release
+REDHAT=/etc/pld-release
 PROC_VERSION=/proc/version
 FPING=@@fping@@
- 
\ No newline at end of file
+
