diff -urN lstat-2.3.2.org/src/misc/lstat.conf.mp.in lstat-2.3.2/src/misc/lstat.conf.mp.in
--- lstat-2.3.2.org/src/misc/lstat.conf.mp.in	2002-11-22 13:11:31.000000000 +0100
+++ lstat-2.3.2/src/misc/lstat.conf.mp.in	2004-12-19 14:42:33.686446968 +0100
@@ -5,8 +5,10 @@
 <Directory "@@WWWDIR@@">
 #for execute CGI files
 Options ExecCGI
-AddHandler perl-script .cgi
-PerlHandler ModPerl::Registry::handler
+# AddHandler perl-script .cgi
+AddHandler cgi-script .cgi
+# PerlHandler ModPerl::Registry::handler
+DirectoryIndex lstat.cgi
 
 AllowOverride AuthConfig Limit
 </Directory>
